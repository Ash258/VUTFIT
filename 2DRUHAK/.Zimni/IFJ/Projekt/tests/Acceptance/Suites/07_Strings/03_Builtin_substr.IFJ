' Substr builtin function

scope
	dim res as string
	dim tmp as integer
	tmp = 0 - 1

	res = substr(!"son hai nguyen", 5, 3)
	print res;

	res = substr(!"son hai nguyen", 9, 120)
	print res;

	res = substr(!"", 1, 1)
	print res;

	res = substr(!"nothing", 0, 1)
	print res;

	res = substr(!"nothing", tmp, 1)
	print res;

	res = substr(!"\nson hai nguyen", 1, tmp)
	print res;

	print !"\nEND";
end scope
