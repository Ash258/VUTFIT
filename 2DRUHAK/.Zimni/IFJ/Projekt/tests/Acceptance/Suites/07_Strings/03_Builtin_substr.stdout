hainguyen
son hai nguyen
END